### DISCUSSION TITLE
Does Organicmaps organization participate in the Google Summer of Code 2024 ?

### DISCUSSION THREAD

**Discussion by tomar19** [organicmaps/organicmaps#6980] [Q&A]
Does the OrganicMaps organization participate in the Google summer of Code 2024? If yes, I'm very excited to join ya'll for the contribution.
Can't wait for it :)   #OrganicmapsGSOC2024 🤘    anyone? @biodranik  @pastk

**Comment by slackuj** [comment]
I would also love to contribute.

**Comment by pastk** [comment]
Please check https://github.com/organicmaps/organicmaps/wiki/GSoC-2024-ideas
